djgentelella/blog/tests/test_entry_detail.py

```diff
@@ -9,7 +9,12 @@
 class TestEntryDetail(TestCase):
 
     def setUp(self):
-        self.entry_1 = models.Entry.objects.create(title=u'Welcome!', is_published=True)
+        self.entry_1 = models.Entry.objects.create(
```

Comment:
For fixtures in test you can use factory-boy it will allow you to easy create fixtures and deal with django model dependencies. https://factoryboy.readthedocs.io/en/stable/